.github/workflows/review.yml

```diff
@@ -139,6 +141,16 @@ jobs:
         run: |
           echo "K8S_SECRET_CONSUMER=1" >> $GITHUB_ENV
 
+      - name: Run migrations setting
+        if: matrix.APPLY_MIGRATIONS == 1
+        run: |
+          echo "K8S_SECRET_APPLY_MIGRATIONS=1" >> $GITHUB_ENV
+
+      - name: Initialize data setting
+        if: matrix.INITIALIZE_DATA == 1
+        run: |
+          echo "K8S_SECRET_INITIALIZE_DATA=1" >> $GITHUB_ENV
+
```

Comment:
Maybe all three above as "Persister settings" (like API setting below)?

Reply:
How about now?

Reply:
I was thinking more of one of where those 3 settings are under.

Reply:
![](https://media.giphy.com/media/l0G18G3m69vQCOddm/giphy.gif)

Maybe like this?

Reply:
Now it's clearer. OFC some if's might be redundant (and just setting the variable would be enough), but now it's at least clear to see that there isn't many common variables.